### PR TITLE
Add suggest-psp blueprint

### PR DESCRIPTION
**Pull request by prachirp**

### CODE REVIEW

File: examples/suggest-psp/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Suggest Pod Security Policy
+
+The `suggest-psp` KRM config function lints pod security policies by suggesting
```
Comment (frankfarzan) [inline examples/suggest-psp/README.md:3]: Use exact k8s kind: `PodSecurtyPolicy`.

File: examples/suggest-psp/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/suggest-psp .
+kpt fn run suggest-psp --results-dir /tmp
+```
+
+## Expected Results
+
+The `--results-dir` flag let us specify a destination to write function results
```
Comment (frankfarzan) [inline examples/suggest-psp/README.md:19]: Remove `--results-dir` flow if not needed to demonstrate this.

File: examples/suggest-psp/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Suggest Pod Security Policy
```
Comment (frankfarzan) [inline examples/suggest-psp/README.md:1]: title is misleading, it's suggesting changes not suggesting a whole resource.

File: examples/suggest-psp/README.md (hunk 2 above)
Comment (prachirp) [reply examples/suggest-psp/README.md:19]: It's needed to show that the issue is with the `spec.allowPrivilegeEscalation` field and that its `suggestedValue` is `false`.

File: examples/suggest-psp/README.md (hunk 1 above)
Comment (prachirp) [reply examples/suggest-psp/README.md]: Done

File: examples/suggest-psp/README.md (hunk 3 above)
Comment (prachirp) [reply examples/suggest-psp/README.md]: Renamed

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**